src/utils/clmfates_interfaceMod.F90

```diff
@@ -2075,151 +2063,29 @@ subroutine init_history_io(this,bounds_proc)
                               default=trim(vdefault),                       &
                               set_lake=0._r8,set_urb=0._r8)
 
-        case(patch_ground_r8)
+        case(patch_ground_r8, patch_size_pft_r8)
+
            d_index = this%fates_hist%dim_kinds(dk_index)%dim2_index
            dim2name = this%fates_hist%dim_bounds(d_index)%name
            call hist_addfld2d(fname=trim(vname),units=trim(vunits),         & ! <--- addfld2d
                               type2d=trim(dim2name),                        & ! <--- type2d
                               avgflag=trim(vavgflag),long_name=trim(vlong), &
                               ptr_patch=this%fates_hist%hvars(ivar)%r82d,    & 
-                              default=trim(vdefault),                       &
-                              set_lake=0._r8,set_urb=0._r8)
+                              default=trim(vdefault))
            
-        case(patch_size_pft_r8)
```

Comment:
Why does so many of these case options go away?

Reply:
After looking at the code,  all of these options actually had the exact same arguments (or arguments that could be defined as variables in the loop).  They are still there, but lumped together into 1 case select statement:

https://github.com/ESCOMP/ctsm/pull/766/files/946519e0e5cfe70e3b094d07d166e32357436cf1#diff-d5aa89e69afc455e4e84461df1cfa3b6R2076